### PR TITLE
(PC-6182) Section Aide

### PR DESCRIPTION
**Pull request by blixit**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-6182

## Checklist

I have:

- [x] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [x] Made sure my feature is working on the relevant real / virtual devices.
- [x] Written **unit tests** for my feature.
- [x] Added a **screenshot** for UI tickets.

## Screenshots

| \*iOS - [Phone name] | \*Android - [Phone name] | Tablet - [Phone name] | Little - [Phone name] |
| -------------------- | :----------------------: | --------------------: | --------------------: |
| image  |                          |                       |                       |

## Deploy hard

If native code (ios/android) was modified, **after** the PR is merged, on the master branch, upgrade the **app version** (+1 patch):

- if you want an hard deployment of the testing environment, use `yarn trigger:testing:deploy`

### CODE REVIEW

File: src/features/profile/pages/Profile.test.tsx [hunk 1]
```diff
@@ -27,13 +28,15 @@ async function renderProfile() {
 }
 
 describe('Profile component', () => {
+  beforeEach(() => {
```
Comment (kopax-polyconseil) [inline src/features/profile/pages/Profile.test.tsx:31]: `beforeEach(navigate.mockRestore)`

File: src/features/profile/pages/Profile.test.tsx [hunk 2]
```diff
@@ -42,7 +45,23 @@ describe('Profile component', () => {
...
     row.props.onClick()
 
-    // TODO: PC-
     expect(navigate).toBeCalledWith('TemporyProfilePage')
   })
+  it('should navigate when the how-it-works row is clicked', async () => {
+    const { getByTestId } = await renderProfile()
+
+    const row = getByTestId('row-how-it-works')
+    row.props.onClick()
```
Comment (kopax-polyconseil) [inline src/features/profile/pages/Profile.test.tsx:54]: Pourquoi on trigger le onClick via la props et pas via `fireEvent` ?

File: src/features/profile/pages/Profile.test.tsx [hunk 3]
```diff
@@ -42,7 +45,23 @@ describe('Profile component', () => {
...
+
+    expect(navigate).toBeCalledWith('TemporyProfilePage')
+  })
+  it('should navigate when the faq row is clicked', async () => {
+    const openExternalUrl = jest.spyOn(NavigationHelpers, 'openExternalUrl')
+    const { getByTestId } = await renderProfile()
+
+    const row = getByTestId('row-faq')
+    row.props.onClick()
```
Comment (kopax-polyconseil) [inline src/features/profile/pages/Profile.test.tsx:63]: idem

File: src/features/profile/pages/Profile.test.tsx (hunk 1 above)
Comment (justinemignot) [reply src/features/profile/pages/Profile.test.tsx:31]: 👍

File: src/features/profile/pages/Profile.tsx [hunk 4]
```diff
@@ -78,3 +99,19 @@ const Section = styled(ProfileSection).attrs({
...
+})
+
+const ExternalLinkSiteIcon = (_props: IconInterface) => (
+  <ExternalLinkSiteContainer>
+    <ExternalLinkSite />
+  </ExternalLinkSiteContainer>
+)
+
+const ExternalLinkSiteContainer = styled.View({
+  width: 32,
```
Comment (justinemignot) [inline src/features/profile/pages/Profile.tsx:113]: utilise getSpacing ici :)

File: src/features/profile/pages/Profile.tsx [hunk 5]
```diff
@@ -78,3 +99,19 @@ const Section = styled(ProfileSection).attrs({
...
+  height: 32,
```
Comment (justinemignot) [inline src/features/profile/pages/Profile.tsx:114]: idem ici (getSpacing)

File: src/features/profile/pages/Profile.tsx [hunk 6]
```diff
@@ -42,16 +47,32 @@ export const Profile: React.FC = () => {
             testID="row-password"
           />
         </Section>
-        <ProfileSection title={_(t`Aides`)}>
-          <Typo.Body>{_(t`Temporary content`)}</Typo.Body>
-        </ProfileSection>
-        <ProfileSection title={_(t`Autres`)}>
+        <Section title={_(t`Aides`)}>
+          <Row
+            title={_(t`Comment ça marche`)}
```
Comment (justinemignot) [inline src/features/profile/pages/Profile.tsx:52]: il manque la modif du fichier messages.po

File: src/features/profile/pages/Profile.test.tsx (hunk 2 above)
Comment (justinemignot) [reply src/features/profile/pages/Profile.test.tsx:54]: 👍

File: src/features/profile/pages/Profile.tsx (hunk 4 above)
Comment (blixit) [reply src/features/profile/pages/Profile.tsx:113]: pourquoi ? c'est ni une marge, ni un padding. C'est une icone et les tailles d'icône sont standard

**Pull request merged**